examples/docker-helloworld.yaml

```diff
@@ -30,6 +30,7 @@ gateway:
     - controller
     - registry
   container_name: gateway
+  command: /usr/bin/run_filebeat.sh
```

Comment:
Sorry for the confusion. we need A8_SERVICE env var for the helloworld python containers. May be that can go as part of the config.yaml for helloworld, illustrating how to set env vars in the command section.

For the gateway, the run_filebeat should be part of its config.yaml right?